server/filters/formatDateFormValue.ts

```diff
@@ -0,0 +1,11 @@
+export default function formatDateFormValue(value: string): string {
+  const targetDateRange = TargetDateRange[value as keyof typeof TargetDateRange]
+  return targetDateRange
+}
+
+export enum TargetDateRange {
```

Comment:
I would be tempted to remove the word "target" to make it a bit more generic (i.e. not just for a step's "target" date range).

Reply:
Agreed; that would be a good refactoring 👍

Reply:
Done in https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/pull/49/commits/6dbed7e2d02d5258ee96d1d53d769fc6bb52d1ba